.bundle/config

```diff
@@ -1,4 +1,2 @@
 ---
-BUNDLE_WITHOUT: omnibus_package
```

Comment:
I thought we wanted this config so we could ensure that initial `bundle installs` by people checking out this repo aren't massive?